### PR TITLE
Pre-Summary Space Option

### PR DESCRIPTION
**Pull request by alecmerdler**
### Description

Adds `--pre-summary-space` option (default `False`) to add a leading space before the triple quotes of a docstring summary.

```python
def foo():
    """ This function does cool things."""
```

Addresses https://github.com/myint/docformatter/issues/39

### CODE REVIEW

**Comment by alecmerdler** [comment]
@myint Does this PR look good to you?

**Comment by alecmerdler** [comment]
@myint Does this PR look good to you?

**Review by myint — DISMISSED** [review]
Seems fine to me! I've not had time to log into `github.com` in quite a while.

Can you take a look at the merge conflict? Thanks!

**Comment by alecmerdler** [comment]
@myint I rebased my PR and fixed the conflict. Look good to you?

**Review by weibullguy — COMMENTED** [review]
See @thejcannon comments in issue #39 regarding black.  Since black is a popular and widely used autoformatter, could you add a section to the README discussing the potential pitfalls of using black and this option together.

File: docformatter.py [hunk 1]
```diff
@@ -629,6 +634,10 @@ def _main(argv, standard_out, standard_error, standard_in):
                         action='store_true',
                         help='add a newline before the summary of a '
                              'multi-line docstring')
+    parser.add_argument('--pre-summary-space',
+                        action='store_true',
+                        help='add a space before the summary of a '
+                             'docstring')
```
[the remark this answers is not in the document]
Comment (weibullguy) [reply docformatter.py]: I added the help string clarification and opened #94 to address the black incompatibility.  I'll add a discussion to the README in an upcoming PR.

**Pull request merged**